src/zjs_spi.c

```diff
@@ -215,7 +224,7 @@ static ZJS_DECL_FUNC(zjs_spi_open)
 
     // Default values
     u32_t bus = 0;
-    double speed = 10;
+    double speed = 2;
```

Comment:
Likely because K64 uses the new SPI API while A101 uses legacy. I'll test out your changes, thanks.